xmppserver/src/main/webapp/muc-room-retirees.jsp

```diff
@@ -0,0 +1,302 @@
+<%@ page contentType="text/html; charset=UTF-8" %>
```

Comment:
We're trying to get away from scriptlets, and use JSTL instead. Could you at the very least condense the Java code to the start of the page, place any variables that you want on the request context, and use those in JSTL in the remainder of the page please?

Reply:
I think I've covered this now.